src/js/edu-benefits/components/military-history/RotcScholarship.jsx

```diff
@@ -2,15 +2,9 @@ import React from 'react';
 
 import ErrorableTextInput from '../../../common/components/form-elements/ErrorableTextInput';
 
-import { isValidYear, validateIfDirty, isNotBlank, isValidMonetaryValue } from '../../utils/validations';
+import { isValidYear, validateIfDirty, isValidMonetaryValue, isBlank } from '../../utils/validations';
 
-function isValidRequiredAmount(val) {
-  return isNotBlank(val) && isValidMonetaryValue(val);
-}
-
-function isValidRequiredYear(val) {
-  return isNotBlank(val) && isValidYear(val);
-}
+const validateIfNotBlank = (validator) => (val) => isBlank(val) || validator(val);
```

Comment:
should we just move this into the validations.js file to import here, if it doesn't already exist there?